code/game/objects/items/cards_ids.dm

```diff
@@ -149,32 +152,33 @@
 			to_chat(user, "You insert [holochip] into [src], adding [holochip.credits] credits to your account.")
 			qdel(holochip)
 
-/obj/item/card/id/attack_self(mob/user)
-	if(!registered_account)
-		var/new_bank_id = input(user, "Enter your account ID.", "Account Reclamation", 111111) as num
-		if(!new_bank_id || new_bank_id < 111111 || new_bank_id > 999999)
-			to_chat(user, "The ID needs to be between 111111 and 999999.")
-			return
-		for(var/A in SSeconomy.bank_accounts)
-			var/datum/bank_account/B = A
-			if(B.account_id == new_bank_id)
-				B.bank_cards += src
-				registered_account = B
-				to_chat(user, "Your account has been assigned to this ID card.")
+/obj/item/card/id/AltClick(mob/living/user)
+	if(Adjacent(user))

```

Comment:
This doesn't make sure the thing isn't a ghost IIRC?

Reply:
welp w.e i added a living check